import/webview/rawwebview.h

```diff
@@ -52,7 +52,7 @@ class RawWebView : public QuickMozView
     void footerMarginChanged();
     void contentOrientationChanged(Qt::ScreenOrientation orientation);
     void acceptTouchEventsChanged();
-    void openUrlInNewWindow();
+    void openUrlInNewWindow(const quint32 &parentId, const uintptr_t &parentBrowsingContext);
```

Comment:
is this signal usually handled by C++ code or by QML code?  If the latter, it might have some issues with the second parameter.

Reply:
This signal is completely wrong. Let's drop all arguments. We cannot really do anything meaningful with this API at the moment. That's also the reason why requested this to internal whilst documenting as this won't work well without parentId either.

Reply:
Did somewhat similar cleanup here: https://github.com/sailfishos/qtmozembed/pull/31/files#diff-334faccf9531e149b1c3bd2583bf513303a0b44e54a3b8c84b5a3a7037abf3fbR87